templates/base.html.twig

```diff
@@ -111,6 +111,8 @@
                             {{ msg('presets') }}:
                             <a href="#" class="preset-link" data-value="js">JS/CSS/JSON</a>
                             &middot;
+                            <a href="#" class="preset-link" data-value="lua">Lua</a>
```

Comment:
I'm assuming Lua (like JS/CSS/JSON) doesn't need to be translated? I can't tell from looking at i18n files in Scribuntu

Reply:
Yeah, not sure about it. I assumed it's the same as JS etc. (Which TBH I am also unsure about - how would they be written in a non-Latin script?)